### PR TITLE
Added suit for HA master.

### PR DESCRIPTION
**Pull request by jszczepkowski**
Added suit for HA master.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/1198)

### CODE REVIEW

**Comment by jszczepkowski** [comment]
CC @wojtek-t @gmarek

**Comment by jszczepkowski** [comment]
@kubernetes/test-infra-maintainers 
I want to add a new suit for HA-master e2e tests. I got the following error:
```
$ go test k8s.io/test-infra/testgrid/config/...
--- FAIL: TestConfig (0.02s)
	config_test.go:122: Testgroup kubernetes-e2e-ha-master - defined but not used in any dashboards
FAIL
FAIL	k8s.io/test-infra/testgrid/config	0.026s
```
Can you help me and tell what I should do?

**Comment by jszczepkowski** [comment]
Part of https://github.com/kubernetes/features/issues/48

**Comment by jszczepkowski** [comment]
@kubernetes/test-infra-maintainers 

The PR is ready for the review.

File: jobs/ci-kubernetes-e2e-gce-ha-master.sh [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+export CLOUDSDK_CORE_PRINT_UNHANDLED_TRACEBACKS="1"
+
+### project-env
+# expected empty
+
+### job-env
+export GINKGO_TEST_ARGS="--ginkgo.focus=\[Feature:HAMaster\]"
+export PROJECT="kubernetes-ha-master"
+# TODO: Enable this when we've split 1.2 tests into another project.
+export FAIL_ON_GCP_RESOURCE_LEAK="false"
```
Comment (gmarek) [inline jobs/ci-kubernetes-e2e-gce-ha-master.sh]: dupe variable

Comment (jszczepkowski) [inline jobs/ci-kubernetes-e2e-gce-ha-master.sh]: removed

**Pull request merged**